modules/assim.sequential/R/downscale_function_hrly.R

```diff
@@ -9,6 +9,7 @@
 #' @param covariates SpatRaster stack, used as predictors in randomForest. Layers within stack should be named. Recommended that this stack be generated using 'covariates' instructions in assim.sequential/inst folder
 #' @return It returns the `downscale_output` list containing lists for the training and testing data sets, models, and predicted maps for each ensemble member.
 #' @import ncdf4
+#' @import lubridate
```

Comment:
Code below looks great. But note that our preference would be to use function namespaces (e.g. ncdf4::nc_open, lubridate::ymd_hm) over `@import`